src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Scanner;
+
 public class Duke {
+    public String greetUser() {
+        return "Hello! I'm Duke" + "\n" + "What can I do for you?";
+    }
+
+    public String farewellUser() {
```

Comment:
More declarative method naming should be used. This method merely generates a farewell message, so `getExitMessage` would be clearer.

Likewise for `greetUser` - it can be clearer if it is named `getWelcomeMessage`